### DISCUSSION TITLE
FullStack.Net = Ao subir a Api Localhost não encontrada

### DISCUSSION THREAD

**Discussion by HeberOliveira15** [balta-io/forum#658] [Tira Dúvidas] [answered]
Boa tarde, @andrebaltieri , tudo bem ?

Dei inicio ao curso de fullstack 2024, adicionei os pacotes e tudo, porem ao subir a APi http, da esse erro no localhost. 
![image](https://github.com/user-attachments/assets/5e9cc748-c55c-45b6-b5b5-93422a8ccbf9)

mesmo com o AddTransient<Handler>() ele continua dando erro.

![image](https://github.com/user-attachments/assets/57df3e35-fe57-4cd2-be7e-8a5fa04cfb0f)

assim está o meu lauchSettingsJson.

![image](https://github.com/user-attachments/assets/418d0ecd-9e3e-4863-b0dd-5e0da0732ab4)

poderiam me ajudar, por favor ?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @HeberOliveira15, como vai?

Você tem alguma rota "/" na aplicação? Pode ser que sua API esteja OK, só não achou a rota raiz.
Tem algum erro no Console?

**Reply by HeberOliveira15** [reply]
Obrigado balta, consegui, estava faltando um "/"
